README.md

```diff
@@ -0,0 +1,97 @@
+# Manual to add openedx_nodebb_discusson in your edx platform
+
+1.  Clone this repo into `openedx_nodebb_discussion` folder using the given command
+
+    `git clone https://github.com/arbisoft/openedx-nodebb-discussion.git openedx_nodebb_discussion`
+
+---
+
+2.  Copy this folder and place this at the following location in edx codebase
+
+    `../edx-platform/openedx/features/`
+
+---
+
+3.  Add `ENABLE_NODEBB_DISCUSSION` in the following files
```

Comment:
This flag should be added under the `FEATURES` key in the settings file

Reply:
Change the relevant code as well.

Reply:
Done